lib/src/observable_map.dart

```diff
@@ -45,7 +45,7 @@ class ObservableMap<K, V> extends Observable implements Map<K, V> {
   ObservableMap() : _map = HashMap<K, V>();
 
   /// Creates a new observable map using a [LinkedHashMap].
-  ObservableMap.linked() : _map = LinkedHashMap<K, V>();
+  ObservableMap.linked() : _map = HashMap<K, V>();
```

Comment:
Does a lint suggest this change? Seems like a breaking change if someone was relying on the insertion order of the linked hash map since it was mentioned in the dartdoc comment.

Reply:
This shouldn't change behavior; the prefer_collection_literals lint gets upset about the old code because LinkedHashMap is the default implementation of HashMap and so it thinks this is overly specific.

If anything, perhaps ObservableMap.linked() should be deprecated since it's the same as the default constructor.